.test-defs/CreateShoot.yaml

```
@@ -8,6 +8,27 @@ spec:
 
   command: [bash, -c]
   args:
-  - go run -mod=vendor ./.test-defs/cmd/create-shoot
+  - >-
+    go test ./test/integration/shoots/creation
```

```suggestion
    go test -mod=vendor ./test/integration/shoots/creation
```
please also readd this flag so we use the vendor folder and do not download the dependencies during the test